pyproject.toml

```diff
@@ -65,7 +65,8 @@ Changelog = "https://github.com/pybamm-team/pybamm-cookiecutter/releases"
 Chen2020 = "pybamm_cookiecutter.parameters.input.Chen2020:get_parameter_values"
 
 [project.entry-points."models"]
-SPM = "pybamm_cookiecutter.models.input.SPM:SPM"
+SPM = "pybamm_cookiecutter.models.input.spm:SPM"
+Reservoir = "pybamm_cookiecutter.models.input.basic_reservoir:BasicReservoir"
```

Comment:
Do we even need these in the project now?

Reply:
I think we could start removing things from the project when things inside the template get stable. At the end of the day, we will deliver the template to the users and not the project, so there is not much of a reason to maintain and test the project, other than for the template generation tests and maybe prototyping.

Reply:
Sounds good, thanks!